packages/kit/src/runtime/server/page/render.js

```diff
@@ -80,12 +80,23 @@ export async function render_response({
 
 		const session = writable($session);
 
+		const updated = (() => {
+			const { set, update, subscribe } = writable(false);
+			return {
+				set,
+				update,
+				subscribe,
+				check: () => {}
+			};
+		})();
```

Comment:
```suggestion
		const updated = readable(false);
		updated.check = () => {};
```

Reply:
we could in fact hoist this outside the function and reuse it, since its value will never change